mica/vv/tests/test_vv.py

```diff
@@ -30,3 +30,9 @@ def test_run_vv():
     obi = process.get_arch_vv(2121)
     assert np.allclose(obi.info()['sim']['max_d_dy'], 0.002197265625)
 
+def test_run_vv_omitted_slot():
+    obi = process.get_arch_vv(19991, version='last')
```

Comment:
That's fine, but put in a code comment to that effect.  Also, if a value like `obi` is not actually used then just skip it.  This makes pyflakes happier.
```
def test_run_vv_omitted_slot():
    process.get_arch_vv(19991, version='last')
```